### PR TITLE
Feature/enc app

### PR DESCRIPTION
**Pull request by jacobberlier**
This adds the ability to deploy encrypted (and unencrypted) apps via cappsd.  See readme for updates to package format.  NOTE: THIS IS A BREAKING CHANGE.  Old package formats will not work.

### CODE REVIEW

**Comment by jacobberlier** [comment]
@jpward

File: handlers/handler.go [hunk 1]
```diff
@@ -305,6 +328,121 @@ func (h *Handler) killApplication(w http.ResponseWriter, r *http.Request) {
 	json.NewEncoder(w).Encode(response)
 }
 
+func (h *Handler) createKey(w http.ResponseWriter, r *http.Request) {
+	response := BasicResponse{Status: Ok, Error: ""}
+	log.Println("Provisioning new decryption key!")
+	//todo: make key intead of touching a file
```
Comment (jpward) [inline handlers/handler.go]: assuming this comment should be removed

File: handlers/handler.go [hunk 2]
```diff
@@ -305,6 +328,121 @@ func (h *Handler) killApplication(w http.ResponseWriter, r *http.Request) {
...
+
+	decoder := json.NewDecoder(r.Body)
+	var nameJson KeyName
+	err := decoder.Decode(&nameJson)
+	if err != nil {
+		log.Printf("Could not proccess request body:\n%v", err)
```
Comment (jpward) [inline handlers/handler.go]: process (ugh i'm correcting spelling forgive me)

File: handlers/handler.go (hunk 1 above)
Comment (jacobberlier) [reply handlers/handler.go]: Yep, I'll fix that tonight or tomorrow

**Comment by jacobberlier** [comment]
How's it looking?

**Comment by jpward** [comment]
The good:
- Builds in yocto fine, including building image into OS at build time
- Starts LMC fine.

The bad:
- LMC is experiencing an error and restarts indefinitely, it doesn't have anything to do with cappsd but I want to get it working before we introduce more changes.

**Pull request merged**